salt/modules/zypper.py

```diff
@@ -357,10 +357,12 @@ def info_installed(*names, **kwargs):
         t_nfo = dict()
         # Translate dpkg-specific keys to a common structure
         for key, value in pkg_nfo.items():
-            if six.PY2 and type(value) == str:
+            if type(value) == str:
```

Comment:
@rallytime Here I am actually not sure if the type is `str` or `bytes` in case of Python3. In case it is `bytes`, that won't work very well. I believe this is wrong and should be something like:
```python
from past.builtins import basestring
...
if isinstance(value, basestring):
   ...
```